tests/functional_tests/t0_main/grpc/t3_use_artifact.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+
+import os
+
+import wandb
+
+# Temporary environment variable for testing grpc service mode
+os.environ["WANDB_SERVICE_TRANSPORT"] = "grpc"
+
+wandb.require("service")
+wandb.init()
```

Comment:
```suggestion
run = wandb.init()
```

minor: but could you use the run object instead we are trying to go with this pattern